lms/djangoapps/teams/views.py

```diff
@@ -310,13 +311,16 @@ def get(self, request):
             result_filter.update({'topic_id': request.QUERY_PARAMS['topic_id']})
         if 'include_inactive' in request.QUERY_PARAMS and request.QUERY_PARAMS['include_inactive'].lower() == 'true':
             del result_filter['is_active']
-        if 'text_search' in request.QUERY_PARAMS:
-            return Response(
-                build_api_error(ugettext_noop("text_search is not yet supported.")),
-                status=status.HTTP_400_BAD_REQUEST
-            )
 
         queryset = CourseTeam.objects.filter(**result_filter)
+        if 'text_search' in request.QUERY_PARAMS:
+            search_engine = CourseTeamIndexer.engine()
+            text_search = request.QUERY_PARAMS['text_search'].encode('utf-8')
+            if search_engine:
```

Comment:
If search is not enabled, do we want to provide any information about the fact that the search term was ignored? At the very least, it seems like we should log this.